citesphere/src/main/java/edu/asu/diging/citesphere/core/service/giles/impl/GilesUploadCheckerImpl.java

```diff
@@ -140,8 +140,23 @@ public void checkUploads() {
                     // Giles is still procoessing
                     logger.debug("Upload " + upload.getProgressId()
                             + " still being processed.");
-                    checkedUploads.add(upload);
-                    continue;
+                    ObjectMapper mapper = new ObjectMapper();
+                    String jsonBody = response.getBody();
+                    GilesCheckUploadResponse processed = new GilesCheckUploadResponse();
```

Comment:
this does not need to be initialized here does it? it either is set in line 147 or processed will never be used?